src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleTypeSymbol.vb

```diff
@@ -664,21 +668,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
 
                                     Dim defaultName As String = TupleMemberName(tupleFieldIndex + 1)
-                                    ' Add a field with default name if the given name Is different
-                                    If namesOfVirtualFields(tupleFieldIndex) <> defaultName Then
-                                        ' The name given doesn't match default name ItemTupleTypeSymbol.RestPosition, etc.
-                                        members.Add(New TupleRenamedElementFieldSymbol(Me, FieldSymbol, defaultName, -members.Count - 1, Nothing))
-                                    End If
 
-                                    ' Add a field with the given name
-                                    Dim location = If(_elementLocations.IsDefault, Nothing, _elementLocations(tupleFieldIndex))
+                                    ' Add a field with default name regardless
+                                    members.Add(New TupleRenamedElementFieldSymbol(Me, FieldSymbol, defaultName, -members.Count - 1, Nothing))
```

Comment:
In this particular case, we are dealing with extended fields (Item8, ...) so the name will never match the name of the underlying field anyways.

Also, this part has significantly changed on the C# side because of the definite assignment work. Instead of "Renamed" we now have "Virtual" fields, because distinction between actual fields and aliases is more significant, than whether the name matches underlying or not.
I will port those changes separately, there should be no impact on the end result of this change or on testcases though.